test/lib/upload-forms.spec.js

```diff
@@ -52,6 +53,8 @@ describe('upload-forms', () => {
       const form = await api.db.get('form:example');
       expect(form.type).to.equal('form');
       expect(form.internalId).to.equal('different');
+      expect(form.xmlVersion.time).to.equal(123123);
+      expect(form.xmlVersion.sha256).to.equal('7e3bb121779a8e9f707b6e1db4c1b52aa6e875b5015b41b0a9115efa2d0de1d1');
```

Comment:
Probably worth leaving a comment in here just to call out that if this assert fails (and you just made changes to the `example.xml`), then this is expected behavior and you can just update the expected has value....